shell/browser/electron_browser_client.cc

```diff
@@ -1017,12 +1018,10 @@ ElectronBrowserClient::GetPlatformNotificationService(
 }
 
 base::FilePath ElectronBrowserClient::GetDefaultDownloadDirectory() {
-  // ~/Downloads
-  base::FilePath path;
-  if (base::PathService::Get(base::DIR_HOME, &path))
-    path = path.Append(FILE_PATH_LITERAL("Downloads"));
-
-  return path;
+  base::FilePath download_path;
+  if (!chrome::GetUserDownloadsDirectory(&download_path))
+    return base::FilePath();
+  return download_path;
```

Comment:
Minor nit, but since the chrome path provider is installed, can we go through that? i.e.

```suggestion
  base::FilePath download_path;
  if (base::PathService::Get(chrome::DIR_DEFAULT_DOWNLOADS, &download_path))
    return download_path;
  return base::FilePath();
```

It should be functionally the same, but it goes through the "public" API instead of using a function from `_internal.h`